lib/cisco_node_utils/interface.rb

```diff
@@ -903,6 +914,444 @@ def switchport_trunk_native_vlan=(val)
       end
     end
 
+    # Interface private vlan configuration properties
+    # private vlan mode (host, promisc, trunk promisc , trunk sec)
+    # private vlan mapping
+    # private vlan native vlan
+    # private vlan allow vlan
+
+    def cli_error_check(result)
```

Comment:
The two methods are not actually same. For interface the error is in body[1] for vln in body[2] and for interfaces we do not have a warning msg